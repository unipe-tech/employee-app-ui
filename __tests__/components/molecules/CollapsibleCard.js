import { render, fireEvent, waitFor } from "@testing-library/react-native";
import { describe, expect, test } from "@jest/globals";
import CollapsibleCard from "../../../components/molecules/CollapsibleCard";

describe("CollapsibleCard", () => {
  test("renders UI correctly", () => {
    const tree = render(<CollapsibleCard />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
