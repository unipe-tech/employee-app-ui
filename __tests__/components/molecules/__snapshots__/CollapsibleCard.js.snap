// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CollapsibleCard renders UI correctly 1`] = `
<View
  style={
    Object {
      "alignSelf": "center",
      "backgroundColor": "#f1f1f1",
      "borderRadius": 10,
      "marginTop": 19.736842105263158,
      "padding": 29.605263157894736,
      "width": "100%",
    }
  }
>
  <View
    style={
      Object {
        "alignItems": "center",
        "flexDirection": "row",
        "width": "100%",
      }
    }
  >
    <View
      style={
        Object {
          "alignItems": "center",
          "flexDirection": "row",
        }
      }
    >
      <Text
        style={
          Object {
            "color": "#2CB77C",
            "fontFamily": "Montserrat-Bold",
            "fontSize": 14,
            "lineHeight": 18,
            "paddingRight": 5,
          }
        }
      />
    </View>
    <Text />
  </View>
  <View
    collapsable={false}
    pointerEvents="none"
    style={
      Object {
        "height": 0,
        "overflow": "hidden",
      }
    }
  >
    <View
      collapsable={false}
      forwardedRef={[Function]}
      onLayout={[Function]}
      style={Object {}}
    >
      <View
        style={
          Object {
            "marginTop": 9.868421052631579,
          }
        }
      />
    </View>
  </View>
</View>
`;
